src/extension.ts

```diff
@@ -407,6 +420,148 @@ async function getDmypyExecutable(folder: vscode.Uri, warnIfFailed: boolean, cur
 	return dmypyExecutable;
 }
 
+async function checkAllNotebooks() {
+	for (const notebook of vscode.workspace.notebookDocuments) {
+		await checkNotebook(notebook)
+	}
```

Comment:
This should be marginally faster given that `Promise.all` can enque all the `Promises` at once on the eventloop.

```suggestion
	await Promise.all(vscode.workspace.notebookDocuments.map(notebook => checkNotebook(notebook)));
```